recipes/install_ext_services.rb

```diff
@@ -43,3 +43,22 @@
         action [:enable, :start]
     end
 end
+
+# Create RabbitMQ user
+execute "create-abiquo-rabbit-user" do
+    command "rabbitmqctl add_user abiquo abiquo"
+    action :nothing
+    subscribes :run, "service[rabbitmq-server]"
+end
+
+execute "set-abiquo-rabbit-user-administrator" do
+    command "rabbitmqctl set_user_tags abiquo administrator"
+    action :nothing
+    subscribes :run, "execute[create-abiquo-rabbit-user]", :immediately
+end
+
+execute "set-abiquo-rabbit-user-permissions" do
+    command 'rabbitmqctl set_permissions -p / abiquo ".*" ".*" ".*"'
+    action :nothing
+    subscribes :run, "execute[create-abiquo-rabbit-user]", :immediately
```

Comment:
Hmmm, this will cause the commands to be executed every time the service "changes". Every restart will trigger this, right?